### ISSUE TITLE
Node v9+ support

### ISSUE THREAD

**Issue by alexcastillo** [urish/node-lsl#1]
Hey Uri!

It seems node-lsl does not support node v9+ because of its `ffi` dependency. It looks they last published a new version [2 years ago](https://www.npmjs.com/package/ffi).

I was looking into other alternatives for node bindings and found:
https://www.npmjs.com/package/fastcall

Dano mentioned Node Native Bridge and WASM.

What are your thoughts?

**Comment by jdpigeon** [comment]
I'm doing a bunch of work on my fork: https://github.com/makebrainwaves/node-lsl

Since fastcall is supposed to be a drop-in replacement for node-ffi I can play around with adding that and, if it works, merge those changes back to this repo.

BTW guys, I think we can make this the official liblsl Node wrapper eventually :fire:

**Comment by alexcastillo** [comment]
Sounds like a plan, go Dano!

**Comment by urish** [comment]
Awesome!

**Comment by CalvinLeather** [comment]
Ended up going the n-api route for ffi and ref, have the results on a fork here: working on node v12 https://github.com/CalvinLeather/node-lsl if anyone wants to use before I clean up and possible PR into this repo.

Is this repo open for PRs? I've been using this repo on a couple projects, wouldn't mind maintaining a bit while I use it.

**Comment by urish** [comment]
@CalvinLeather yes, I can add you as a collaborator if you are open to maintaining it

**Comment by CalvinLeather** [comment]
Sounds good, yeah I will be working with it for the next year or so at least. I'll PR in a bit. Thanks!

**Comment by oori** [comment]
@CalvinLeather any plan to push your PR ?      would be nice to have this update on npm.
